tmdrv_devices/thrustmaster_tmx.py

```diff
@@ -0,0 +1,9 @@
+name = 'Thrustmaster TMX'
+idVendor = 0x044f
+idProduct = [0xb67e, 0xb65d, 0xb67f]
+control = [
+	{'step':1, 'request_type':0x41, 'request':83, 'value':0x0001, 'index':0x0000, 'data':b''},
+	{'step':2, 'request_type':0x41, 'request':83, 'value':0x0007, 'index':0x0000, 'data':b''},
+]
+jscal = '6,1,0,32505,32505,16574,2049063,1,0,519,519,1036399,1067305,1,0,512,512,1050596,1054724,1,0,511,511,1050596,1048544,1,0,0,0,536854528,536854528,1,0,0,0,536854528,536854528'
+dev_by_id = 'usb-Thrustmaster_Thrustmaster_TMX_Racing_Wheel-joystick'
```

Comment:
I pulled from upstream to test this change. It doesn't seem to have fixed the issue. I added a print(handle) at line 71 just before connected=true. Both handles from step 1 and 2 initialize and the handle populates, but the jscal line still runs before the symlink populates. I crudely added a sleep(5) and it resolved the issue entirely, which works but obviously isn't the best method. I'd like to have a more technical answer to be honest. I suspect that the device sends some sort of URB type message to the system that it has completed initialization perhaps. I looked around in the pcap data and there are some interesting packets around 966, but I have very little understanding of what is actually happening in them.